_sass/blocks/jekyll-layouts/_state-pages.scss

```diff
@@ -25,18 +22,24 @@
     padding-bottom: $standard-padding-lite;
   }
 
-  .title-land-ownership:not(.chart-title):not(.state-page-nav-title) {
-    border-bottom: 1px solid $neutral-gray;
-    margin-bottom: $base-padding-lite * 2;
+  h3:not(.chart-title):not(.title-land-ownership) {
+    padding-top: 4rem;
+  }
+
+  ul + h3:not(.chart-title):not(.title-land-ownership),
+  p + h3:not(.chart-title):not(.title-land-ownership) {
+    margin-top: -0.75rem; //to compensate for margin from preceding selectors
   }
 
   h4:not(.chart-title):not(.state-page-nav-title):not(.details-container) {
     border-bottom: 1px solid $neutral-gray;
     font-weight: $weight-light;
+    padding-top: 2.5rem;
```

Comment:
We need the specific values. I started making variables, and then realized we weren't using them anywhere else so it didn't seem worth it. What do you think?